signac/contrib/hashing.py

```diff
@@ -1,14 +1,29 @@
 # Copyright (c) 2017 The Regents of the University of Michigan
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
+"""Hashing functions."""
+
 import hashlib
 # we need to use the standard module here to ensure
 # exact consistent formatting
+
 import json
 
 
 def calc_id(spec):
-    "Calculate and return a hash value for the given spec."
+    """Calculate and return a hash value for the given spec.
+
+    Parameters
+    ----------
+    spec : Collection
+        A JSON-encodable collection.
```

Comment:
This computes a hash for a signac state point - not a signac Collection.
```suggestion
    spec : dict
        A JSON-encodable mapping.
```